child_update_picture_date/wizard/__init__.py

```diff
@@ -0,0 +1,12 @@
+# -*- encoding: utf-8 -*-

```

Comment:
Ok. But then why setup a cron job to perform the task once? You could add a button to the child tree view header to launch the task manually